topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: tutorial_hands_on
+
+title: Designing plasmids encoding predicted pathways by using the BASIC assembly method
+zenodo_link: https://zenodo.org/record/6123887#.YhTCPhvjJH4
+questions:
+- How to design plasmids encoding predicted pathways using BASIC assembly method ?
+- How to generate scripts enabling the automatic build of plasmids ?
+objectives:
+- Annotate the SBML pathway using Selenzy tool.
+- Design plasmids encoding predicted pathways using rpbasicdesign tool.
+- Generate dnabot scripts enabling the automatic build of plasmids.
+time_estimation: 15 min
+key_points:
+- Annotation of SBML pathway using Selenzyme
+- Generating Constructs using rpbasicdesign
+- Generating python scripts using dnabot
+contributors:
+- Kenza BAZI KABBAJ
+- Thomas DUIGOU
+- Joan HERISSON
+- Guillaume GRICOURT
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Synthetic biology is a novel engineering discipline which requires computational tools for the design of metabolic pathways for the production of chemicals.
+
+In this tutorial, we will use a set of tools from the **Genetic Design - BASIC Assembly Workflow** (https://galaxy-synbiocad.org) which will enable you to design plasmids implementing metabolic pathways for the bioproduction of lycopene in _E.coli_ using the [BASIC assembly method](https://doi.org/10.1021/sb500356d), and to generate scripts enabling the automatic build of these plasmids using an Opentrons liquid handler robot. After downloading these scripts onto a computer connected to an Opentrons, one can perform the automated construction of the plasmids at the bench.
```

Comment:
The DOI you used here indicates this might be a good place to use a reference.
```suggestion
In this tutorial, we will use a set of tools from the **Genetic Design - BASIC Assembly Workflow** (https://galaxy-synbiocad.org) which will enable you to design plasmids implementing metabolic pathways for the bioproduction of lycopene in _E.coli_ using the BASIC assembly method (as described in {% cite ... %}), and to generate scripts enabling the automatic build of these plasmids using an Opentrons liquid handler robot. After downloading these scripts onto a computer connected to an Opentrons, one can perform the automated construction of the plasmids at the bench.
```

and then using https://doi2bib.org to get the bibtex entry for that DOI and adding it to the tutorial.bib

Reply:
Ok, done.